ddl/failtest/fail_db_test.go

```diff
@@ -336,10 +336,14 @@ func (s *testFailDBSuite) TestAddIndexWorkerNum(c *C) {
 	start := -10
 	num := 4096
 	// first add some rows
+	dml := "insert into t1 test_add_index"
```

Comment:
```suggestion
	dml := "insert into t1 test_add_index values"
```
Please fix this typo